source/module_io/write_Vxc_lip.hpp

```diff
@@ -0,0 +1,250 @@
+#pragma once
```

Comment:
I do not recommend using #pragma once, I would suggest using #ifdef

Reply:
I suggest change "Vxc" to "vxc"

Reply:
> I suggest change "Vxc" to "vxc"

Using capital letter is to distinguish wavefunction representation (nbands*nbands) from real-space grid representation (nrxx).